Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -1095,6 +1095,68 @@ std::optional<ExactTime> parseInstant(StringView string)
     });
 }
 
+uint8_t dayOfWeek(PlainDate plainDate)
+{
+    Int128 dateDays = static_cast<Int128>(dateToDaysFrom1970(plainDate.year(), plainDate.month() - 1, plainDate.day()));
+    int wd = static_cast<int>((dateDays + 4) % 7);
```

Comment:
`wd` is confusing, but I think this could just be `result`?

Reply:
Or `weekDay`?

Where does the +4 come from? Is that because there were 4 days before a Monday on 1/1/1970?